### PR TITLE
graphia: 2.2 -> 3.0

### PR DESCRIPTION
**Pull request by onny**
###### Description of changes

update and fix build for package `graphia`

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -17713,7 +17713,7 @@ with pkgs;
 
   ghcid = haskellPackages.ghcid.bin;
 
-  graphia = libsForQt5.callPackage ../applications/science/misc/graphia { };
+  graphia = libsForQt514.callPackage ../applications/science/misc/graphia { };
```
Comment (AndersonTorres) [inline pkgs/top-level/all-packages.nix:17716]:
```suggestion
  graphia = libsForQt514.callPackage ../applications/science/misc/graphia { 
    stdenv = gcc10Stdenv;
  };
```

**Review by AndersonTorres — APPROVED** [review]

**Comment by onny** [comment]
Thank you @AndersonTorres, fixed it :)

File: pkgs/applications/science/misc/graphia/default.nix [hunk 2]
```diff
@@ -1,22 +1,39 @@
-{ stdenv, lib, cmake, fetchFromGitHub
-, wrapQtAppsHook, qtbase, qtquickcontrols2, qtgraphicaleffects
+# Using gcc 10 because this fails to build with gcc 11
```
Comment (AndersonTorres) [inline pkgs/applications/science/misc/graphia/default.nix:1]: I believe this comment should be in `all-packages.nix`.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

**Comment by onny** [comment]
Thank you, fixed :)

**Comment by AndersonTorres** [comment]
ofborg complains with `file breakpad-sigstksz.patch not found`.

**Comment by onny** [comment]
@AndersonTorres ups, fixed now :)

**Pull request merged**